### PR TITLE
Clarify Set up Instructions in README

### PR DESCRIPTION
**Pull request by Don-Assamongkol1**
Hey there!

First, I wanted to say thanks so much for releasing this software. I love the functionality to redirect my Notion page to my custom domain. 

Maybe it was just me, but I ran into trouble setting this up. I git cloned notehost because I found this project on Github, and didn't realize that notehost is packaged as a node module that you can execute. In other words, I cloned this repo, installed everything, and kept running `npx notehost init <domain>` not realizing that you're supposed to run `npx notehost init <domain>` after `npm i notehost`. 

Hopefully this helps someone else if they come across this wonderful repo. Whether or not you accept the change, thanks for creating this!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +42,10 @@
 
 ---
 
-Go into your working directory and run:
+Go into the working directory from which you'd like to keep the script to apply the Cloudflare worker configuration and run:
 
 ```sh
+npm install notehost
```
Comment (velsa) [inline README.md]: It is not necessary to install notehost package in order to run it with npx.

The way npx works is - it checks if you have the package already installed and if not - it will install it automagically before running it. 

As for the rest of the changes to the README - I am happy to accept them :)

Comment (Don-Assamongkol1) [reply README.md]: Oh I see-- didn't know npx did that!

Thanks velsa, best of luck for this project.

**Comment by ryeones** [comment]
I'm following the instructions but can't seem to get it right... have little to no knowledge about coding, anyone kind enough to walk me through or explain some of the steps in layman terms

**Comment by DudeThatsErin** [comment]
> I'm following the instructions but can't seem to get it right... have little to no knowledge about coding, anyone kind enough to walk me through or explain some of the steps in layman terms

I have a tutorial up! https://dudethatserin.notion.site/notehost?pvs=4
